includes/class/heygov-resource.php

```diff
@@ -49,5 +50,44 @@ public function load_apps_banner() {
 			<?php
 		}
 	}
+	function heygov_forms_shortcode( $atts = array()) {
+		$args = shortcode_atts( array(
+			'container-small' => false,
+			'department' => 'all'
+		), $atts );
+
+		$small = $args['container-small']; 
+		$department = $args['department'];
+
+        ob_start(); 
+        // HeyGov ID
+        $heygov_id = get_option('heygov_id');
+
+       // Get any existing copy of our transient data
+        if ( false === ( $forms = get_transient( 'forms' ) ) ) {
+            // It wasn't there, so regenerate the data and save the transient
+			$forms = wp_remote_get('https://heygov-api-develop-nxb3467cgq-uc.a.run.app/'.$heygov_id.'/forms?status=public&expand=department&department='.$department);
```

Comment:
the `department` param should not be used in api request (probably doesn't even work).

Filtering should take place just before the including of view file, so at line 77 from how it looks now. 
It should be something like 
```
if (department attribute is set && has value) {
  forms = forms.filter(form => form.department_id === department || form.department.name === department || form.department.slug === department)
}
```